storage/scylladb_identity.h

```diff
@@ -23,6 +23,8 @@ typedef UT_array db_identity_array_t;
 
 typedef enum { PENDING_TXN = 0, INSERTING_TXN, CONFIRMED_TXN, NUM_OF_TXN_STATUS } db_txn_status_t;
 
+#define DB_NUM_TRYTES_HASH NUM_TRYTES_HASH
+
```

Comment:
Why do we need an extra macro here?

Reply:
It would be more flexible if we want to adjust the length.